zcash_primitives/src/legacy/keys.rs

```diff
@@ -405,6 +413,39 @@ impl private::SealedChangeLevelKey for InternalIvk {
 
 impl IncomingViewingKey for InternalIvk {}
 
+/// An incoming viewing key at the "ephemeral" path
+/// `m/44'/<coin_type>'/<account>'/2`.
+///
+/// This allows derivation of ephemeral addresses for use within the wallet.
+#[derive(Clone, Debug)]
+pub struct EphemeralIvk(ExtendedPublicKey<PublicKey>);
+
+impl private::SealedChangeLevelKey for EphemeralIvk {
+    const SCOPE: TransparentKeyScope = TransparentKeyScope(2);
```

Comment:
This looks like the constant that I was asking be moved here from `zcash_client_sqlite`, but it's not public.

Reply:
We now have a public constant `TransparentKeyScope::EPHEMERAL`.

Reply:
In any case this impl is not needed. I will remove it.

Reply:
Removed.